src/targets/javascript/axios/client.ts

```diff
@@ -89,6 +99,6 @@ export const axios: Client = {
     push('console.error(error);', 2);
     push('});', 1);
 
-    return join();
+    return join().replace(/'encodedParams,'/, 'encodedParams,');
```

Comment:
it feels a little hacky and disjointed to do this replace here at the very end.  any way to do it up above?

Reply:
Unfortunately not because encodedParams in the request options needs to be a variable in the snippet, and we can only do that handling after requestoptions had been added to the snippet.

Reply:
I just pushed 2343bb2736e2f23b1db718eb29a970d1c30a740d.  check it out and lemme know what you think of it.

Reply:
Oh I love it. Will update this to use it.

Reply:
Love it.